velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -536,6 +538,30 @@ std::vector<exec::Split> AggregationFuzzerBase::makeSplits(
   return splits;
 }
 
+void AggregationFuzzerBase::Stats::updateReferenceQueryStats(
+    AggregationFuzzerBase::ReferenceQueryErrorCode errorCode) {
+  if (errorCode == ReferenceQueryErrorCode::kReferenceQueryFail) {
+    ++numReferenceQueryFailed;
+  } else if (errorCode == ReferenceQueryErrorCode::kReferenceQueryUnsupported) {
+    ++numReferenceQueryNotSupported;
+  }
```

Comment:
nit: maybe we should have a default catch all if both of these conditions are unsatisfied ?

Reply:
I think numReferenceQueryFailed and numReferenceQueryNotSupported are the only two stats to be updated here. Do you suggest to add an else-branch with an empty body here?